test/integration/tasks.test.js

```diff
@@ -409,7 +414,7 @@ describe("Tasks", function () {
           matchingTasks.forEach((task) => {
             expect(task.title.toLowerCase()).to.include(searchTerm.toLowerCase());
           });
-          expect(matchingTasks).to.have.length(4);
+          expect(matchingTasks).to.have.length(6);
```

Comment:
can we add test to check that the sorting is working as we have changed it to work